client/src/locales/fr-FR.json

```diff
@@ -510,10 +510,19 @@
         },
         "ImportFile": {
             "title": "Importation",
+            "waiting": "en attente",
             "cancel": "annulé",
             "failed": "échoué",
             "failedDetails": "Please check you internet connection.",
```

Comment:
Could you also change the `failedDetails` text? I know it's not related to your changes but may as well fix it.

Reply:
Done